src/installer.js

```diff
@@ -54,8 +54,13 @@ class DebianInstaller extends common.ElectronInstaller {
   /**
    * Copy the application into the package.
    */
-  copyApplication () {
-    return super.copyApplication(src => src !== path.join(this.options.src, 'LICENSE'))
+  async copyApplication () {
+    await super.copyApplication(src => src !== path.join(this.options.src, 'LICENSE'))
+    
+    // The sandbox helper executable must have the setuid (+s / 0o4000) bit set.
+    const sandboxHelperPath = path.join(this.stagingDir, 'chrome-sandbox')
+    if (await fs.exists(sandboxHelperPath))
```

Comment:
`fs.exists` is deprecated, I think.

```suggestion
    if (await fs.pathExists(sandboxHelperPath))
```

Reply:
`pathExists` doesn't seem to exist (ha). https://nodejs.org/api/fs.html#fs_fs_exists_path_callback suggests `fs.access()` is the replacement.